Gemfile

```diff
@@ -5,50 +5,33 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.1.6', '>= 5.1.6.1'
-# Use postgresql as the database for Active Record
+gem 'slim'
```

Comment:
I hope that's a good practice to write versions of gems in Gemfile. Think of using ~> matching in versioning.